src/presets/hundo_data.asm

```diff
@@ -2819,9 +2857,10 @@ preset_hundo_ridley_kihunter_shaft_up:
     dw #preset_hundo_ridley_wasteland_up ; Ridley: Wasteland Up
     dl $7E078D : db $02 : dw $9A3E ; DDB
     dl $7E079B : db $02 : dw $B5D5 ; MDB
-    dl $7E090F : db $02 : dw $0000 ; Screen subpixel X position.
     dl $7E0911 : db $02 : dw $0500 ; Screen X position in pixels
-    dl $7E0913 : db $02 : dw $EC01 ; Screen subpixel Y position
+    dl $7E0913 : db $02 : dw $EC02 ; Screen subpixel Y position
+    dl $7E0917 : db $02 : dw $03C0 ; Layer 2 X scroll in room in pixels?
+    dl $7E0921 : db $02 : dw $03C0 ; BG2 X scroll offset due to room transitions (Translates between Layer 2 scroll and BG2 scroll)
     dl $7E09C2 : db $02 : dw $03F3 ; Health

```

Comment:
I think firefleas preset is also broken, I just noticed it last night

Reply:
I put a fix for that one in scrolls, line 434. Looks like I forgot to comment the room name. I'll add that in.